# 状态标志位
int32 left_connection    # 连接状态
int32 left_lock_state    # 锁定状态
int32 right_connection    # 连接状态
int32 right_lock_state    # 锁定状态
int32 system_status       # 系统状态机状态